### PR TITLE
update collectd to 5.5.0, add uptime, swap, network

### PR DESCRIPTION
**Pull request by mbessler**
- update collectd to version 5.5.0 by updating base image from precise to xenial
- cpu plugin ReportByCpu config added, controlled via env var REPORT_BY_CPU (defaults to false)
- network interface plugin added, configured to ignore loopback and docker related interfaces (lo, docker*, veth*)
tested against KairosDB backend.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,11 +1,11 @@
-FROM    ubuntu:precise
+FROM    ubuntu:xenial
 
 ENV     DEBIAN_FRONTEND noninteractive
 
 # only later versions of collectd have the graphite plugin
-RUN     echo "deb http://us.archive.ubuntu.com/ubuntu/ precise universe" >> /etc/apt/sources.list
-RUN     echo "deb http://ppa.launchpad.net/vbulax/collectd5/ubuntu precise main" >> /etc/apt/sources.list
-RUN     apt-key adv --recv-keys --keyserver keyserver.ubuntu.com 232E4010A519D8D831B81C56C1F5057D013B9839
+#RUN     echo "deb http://us.archive.ubuntu.com/ubuntu/ precise universe" >> /etc/apt/sources.list
```
Comment (andreasjansson) [inline Dockerfile]: Can we delete these lines instead of commenting?

**Comment by andreasjansson** [comment]
Thanks for this too! Just a minor comment.

File: Dockerfile (hunk 1 above)
Comment (mbessler) [inline Dockerfile]: yes definitely, give me just a minute

**Comment by mbessler** [comment]
Dockerfile cleaned

**Pull request merged**